teuthology/task/install/rpm.py

```diff
@@ -148,48 +146,37 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
         _yum_set_check_obsoletes(remote)
 
     if dist_release in ['opensuse', 'sle']:
-        remote.run(
-            args=[
-                'sudo', 'zypper', 'clean', '-a',
-            ])
+        remote.run(args='sudo zypper clean -a')
     else:
-        remote.run(
-            args=[
-                'sudo', 'yum', 'clean', 'all',
-            ])
+        remote.run(args='sudo yum clean all')
 
     ldir = _get_local_dir(config, remote)
 
     if dist_release in ['opensuse', 'sle']:
-        pkg_remove_cmd = 'sudo zypper -n remove --capability {pkg}'
-        pkg_install_cmd = 'sudo zypper -n --no-gpg-checks install --capability --no-recommends {pkg}'
+        remove_cmd = 'sudo zypper -n remove --capability {pkg}'
```

Comment:
Here and below, do we really need this {pkg} template here?

Reply:
Oh... I see, actually, the code is great.

Reply:
we need to handle two cases

1. local package where `{ldir}/{cpack}` is the first choice, and `{cpack}` is an alternative, 
1. and remote package, where `{ldir}` is `None` .